### DISCUSSION TITLE
Could this run on Cloudflare edge workers?

### DISCUSSION THREAD

**Discussion by heathdutton** [bluesky-social/atproto#112] [How to do it?] [answered]
...perhaps with some modification?
Or is that a silly question.

Just curious.

**Comment by dholms** [comment, marked as answer]
I don't see why not. I'm not terribly familiar with the specifics of cloudflare workers, but on the face of it I don't see why not.

You can't just outright stick the whole server in there 😛 but pieces of it could live there.

I'm not sure it makes sense for a user's personal data server to be an edge worker, but I could see indexing servers running some of their services at the edge, for faster updates/aggregation.

What do you have in mind?

**Reply by heathdutton** [reply]
That is about what I was thinking. Thanks for the reply.

The main reason I'm thinking about this is it's an inexpensive route to global availability.
